libbeat/scripts/Makefile

```diff
@@ -360,14 +360,18 @@ docs:  ## @build Builds the documents for the beat
 docs-preview:  ## @build Preview the documents for the beat in the browser
 	PREVIEW=1 $(MAKE) docs
 
-### KIBANA FILES HANDLING ###
+### SETUP commands ###
 ES_URL?=http://localhost:9200
 KIBANA_URL?=http://localhost:5601
 
 .PHONY: import-dashboards
 import-dashboards: update ${BEAT_NAME}
 	${BEAT_GOPATH}/src/${BEAT_PATH}/${BEAT_NAME} setup -E setup.dashboards.directory=${PWD}/_meta/kibana.generated -E setup.kibana.host=${KIBANA_URL} --dashboards
 
+.PHONY: register-pipelines
+register-pipelines: update ${BEAT_NAME}
+	${BEAT_GOPATH}/src/${BEAT_PATH}/${BEAT_NAME} setup --pipelines
```

Comment:
Have you tested this command? I don't think that it's going to upload anything to Elasticsearch with this format.
`setup --pipelines` only uploads piplines of enabled modules. In order to upload something it needs you need to select which pipelines you want to upload either by specifying the module names via `-modules` or enabling them in the configuration.

Reply:
I have tested it for APM Server, and thought it might be worth to add it to libbeat. However, this sounds like it would need to be handled differently, so I can just move the `register-pipelines` cmd to APM Server code instead.